healthcheck/client_test.go

```diff
@@ -0,0 +1,59 @@
+package healthcheck
+
+import (
+	"errors"
+	"net/http"
+	"testing"
+
+	"github.com/ONSdigital/go-ns/healthcheck/mock_healthcheck"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestUnitHealthcheckClient(t *testing.T) {
+
+	service := "myService"
+
+	Convey("Healthcheck returns nil when no error", t, func() {
+		mock := &mock_healthcheck.HttpClientMock{
+			GetFunc: func(url string) (*http.Response, error) {
+				return &http.Response{StatusCode:http.StatusOK}, nil
+			},
+		}
+
+		client := healthcheckClient{Client:mock, Url:"http://foo/bar", Service: service}
+
+		s, e := client.Healthcheck()
```

Comment:
It is certainly conventional to have short variable names for locally scoped variables, however I would consider using `err` for error variables. This seems to be widely used.